### ISSUE TITLE
"interconnect error: connect to api.opensuse.org:443: Network is unreachable" error message using web proxy with authentication.

### ISSUE THREAD

**Issue by DriverXavGeForce** [openSUSE/open-build-service#10956]
# Issue Description
"interconnect error: connect to api.opensuse.org:443: Network is unreachable" error message using web proxy with authentication.

Expected Result
============
Connect to api.opensuse.org with web proxy without error.

How to Reproduce
===============
1. Configure proxy with authentication on OBS server like:

In /etc/sysconfig/proxy:
```
PROXY_ENABLED="yes"
HTTP_PROXY="http://http.internettmp.inettmp.com:80/"
HTTPS_PROXY="http://http.internettmp.inettmp.com:80/"
FTP_PROXY="http://http.internettmp.inettmp.com:80/"
GOPHER_PROXY=""
SOCKS_PROXY=""
SOCKS5_SERVER=""
NO_PROXY="localhost,127.0.0.1,.inettmp.com"
```

In /root/.curlrc
```
# Changed by YaST2 module proxy 03/29/21
--proxy "http://user:******@http.internettmp.inettmp.com:80"
--noproxy "localhost,127.0.0.1,.inettmp.com"
```

In BSConfig.pm
```
our $proxy = "http://user:******\@http.internettmp.inettmp.com:80/";
our $noproxy = "localhost, 127.0.0.1, .inettmp.com";
```

In options.yml
```
# Proxy Configuration
http_proxy: http://user:******@http.internettmp.inettmp.com:80/
https_proxy: http://user:******@http.internettmp.inettmp.com:80/
no_proxy: localhost, 127.0.0.1, .inettmp.com
```

2. Start to build package for SLES 15 SP2 with interconnect activated.

3. Result in log file:
Errors in scheduler_x86_64.log file:
```
2021-03-29 10:18:54: [2473]  looking at high prio TMP:BRE_Lite/SLE_15_SP2 (1/0/5/0/5)
  - TMP:BRE_Lite/SLE_15_SP2
    scanning repo TMP:BRE_Lite/SLE_15_SP2...
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/standard
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Basesystem
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Containers
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Desktop-Applications
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Development-Tools
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Development-Tools-OBS
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-HPC
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Legacy
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Live-Patching
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Public-Cloud
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Python2
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-RT
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Server-Applications
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Web-Scripting
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Module-Transactional-Server
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLES
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Manager-Tools
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Product-HA
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP2:GA/SLE-Product-WE
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP1:Update/config
    fetching remote repository state for openSUSE.org:SUSE:SLE-15-SP1:GA/config
    sorting 1 packages
    checking packages
      - TMP_op (interconnect error: connect to api.opensuse.org:443: Network is unreachable)
    broken: 1
    building: 0, notready: 0, unfinished: 0
    took 0 seconds to check the packages
    leaf prp, freeing data
```

Further Information
===============
OBS Server 2.10.10

**Comment by dmarcoux** [comment]
This error happens in the backend. I don't know though if it's a misconfiguration or not. @mlschroe or @adrianschroeter, could you have a look please?

**Comment by adrianschroeter** [comment]
this will be fixed in 2.10.18
